app/build.gradle

```diff
@@ -6,7 +6,7 @@ apply plugin: 'com.hiya.jacoco-android'
 apply plugin: "com.starter.easylauncher"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
Doesnt updating this have secondary effects?

Reply:
> Doesnt updating this have secondary effects?

Yes, I do feel the same.